### DISCUSSION TITLE
Welcome to TachiyomiAT Discussions!

### DISCUSSION THREAD

**Discussion by mannu691** [mannu691/TachiyomiAT#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mirzabaig313** [comment]
In pre-releases, translation got error and not working - It's Japanese Manga 

Suggestion - You Can Add Sync with Cloud

**Reply by mannu691** [reply]
Can you please provide more details for the error? , is the translation blank or a error while translating?

Please share a screenshot of your settings

**Reply by Mirzabaig313** [reply]
I Provided you Screen-Recording 
, i tried with Chinese manhua its translate but text position not right 

https://github.com/user-attachments/assets/7c663cd7-84c1-489a-bdfe-85bab9f76f6c


It's Google Translator

**Reply by Mirzabaig313** [reply]
I am also using Gemini to translate 

Screenshots of Gemini Ai

![Screenshot_2025-02-24-20-35-55-027_app.kanade.tachiyomi.at.jpg](https://github.com/user-attachments/assets/68641198-ad72-4217-bf85-2d1fb6d17a9b)

![Screenshot_2025-02-24-20-35-57-532_app.kanade.tachiyomi.at.jpg](https://github.com/user-attachments/assets/d2e25d24-a7a1-4d07-8d17-2609fe503497)

![Screenshot_2025-02-24-20-35-52-193_app.kanade.tachiyomi.at.jpg](https://github.com/user-attachments/assets/6357d8ce-ff0b-41ea-97ca-4e3d4073d5d6)

**Reply by Mirzabaig313** [reply]
When I set language Japanese  he translate and If I  set Chinese I got translation error 


![Screenshot_2025-02-24-20-38-07-050_app.kanade.tachiyomi.at.jpg](https://github.com/user-attachments/assets/90dfc40e-684f-4276-8973-bf4a39597e0e)

In this image you can see red color 

![Screenshot_2025-02-24-20-39-50-915_app.kanade.tachiyomi.at.png](https://github.com/user-attachments/assets/202e18f1-eaed-4a1c-aec8-9dafda7cd22b)

**Reply by mannu691** [reply]
I'll take a look , meanwhile please continue any further discussion in
https://github.com/mannu691/TachiyomiAT/issues/18

**Reply by Mirzabaig313** [reply]
I like this app and waited for updates, I Contined to  Discusses and test and tell you.

- I Got One More Error If I Install Extension manualy They not showing in source section

**Reply by mannu691** [reply]
Thanks i appreciate it, 

I don't think anything I changed in the fork would create a problem like that , maybe its the filters?

**Reply by Mirzabaig313** [reply]
What you mean by Filter's

**Reply by mannu691** [reply]
> What you mean by Filter's

Please state which extension you installed in https://github.com/mannu691/TachiyomiAT/issues/19
I'll explain the filters there

**Reply by Mirzabaig313** [reply]
These Chinese Manhua extension,  I install separately  one by one apk.
They are not in extension repo so I install them separately.
After Installation they Not showing in **source** section

 ![IMG_20250226_122504.jpg](https://github.com/user-attachments/assets/b745a514-d9b5-4716-a1c0-33a93630bad8)


I am using these 
![Screenshot_2025-02-26-12-28-36-743_app.kanade.tachiyomi.at.png](https://github.com/user-attachments/assets/186c70db-944c-47a8-9ced-ac67911b3bbd)

**Comment by drewbitt** [comment]
Check out https://openrouter.ai/docs/features/structured-outputs sometime. Noticed many models don't work because they don't give back correct json with your prompt

**Reply by Mirzabaig313** [reply]
It's text overly problem

**Reply by mannu691** [reply]
yes , that is a problem i am aware of. its just that i didn't bother to implement it properly :D
will try it next release 👍

**Reply by Mirzabaig313** [reply]
Do  fast bro , I am waiting . I read manhwa and manhua min 4 hours daily 🥲 .I want translation option they released very slow
